apps/src/lib/kits/maker/ui/setup-instructions.module.scss

```diff
@@ -0,0 +1,24 @@
+:export {
+  checklistId: connectYourBoardChecklist; 
+  installInstructionsId: installInstructions;
+}
+
+.listWithImage {
+  display: grid;
+  grid-template: 1fr / 2fr 1fr;
+  justify-items: center;
+  align-items: center;
+  column-gap: 10px;
+}
+
+.imageContainer img {
+  max-height: 160px;
+}
+
+li p {
+  margin-bottom: 0;
+}
+
+.troubleshootList {
+  list-style-type: "⚠️";
```

Comment:
Could we add a space in-between the warning emoji and text?

Reply:
Oh yeah it is real squished looking. Good catch.